remotespark/utils/eventshandler.py

```diff
@@ -0,0 +1,13 @@
+from remotespark.utils.log import Log
+
+
+class EventsHandler:
+    def __init__(self):
+        self.logger = Log("EventsHandler")
+
+    def store_event(self, *args):
+        """
+        Storing the Event details using the logger.
+        """
+        event_line = ",".join("{}".format(arg) for arg in args)
```

Comment:
You could have sorted key,value pairs. Just don't use a dictionary. Or don't sort them and iterate over them; what's wrong with that?

I'm not focusing on implementation, I'm focusing on building a robust API that any system could use.